SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -697,14 +697,7 @@ bool USpatialSender::UpdateEntityACLs(AActor* Actor, Worker_EntityId EntityId)
 	WorkerAttributeSet OwningClientAttribute = { OwnerWorkerAttribute };
 	WorkerRequirementSet OwningClientOnly = { OwningClientAttribute };
 
-	if (EntityACL->ComponentWriteAcl.Contains(Info->RPCComponents[RPC_Client]))
-	{
-		EntityACL->ComponentWriteAcl[Info->RPCComponents[RPC_Client]] = OwningClientOnly;
-	}
-	else
-	{
-		EntityACL->ComponentWriteAcl.Add(Info->RPCComponents[RPC_Client], OwningClientOnly);
-	}
+	EntityACL->ComponentWriteAcl.Add(Info->RPCComponents[RPC_Client], OwningClientOnly);
```

Comment:
Drive by clean up change. `Add` overwrites an existing Value is the Key already exists.